### PR TITLE
fix slot types on new foci created items

### PR DESCRIPTION
**Pull request by ThyWoof**

### CODE REVIEW

**Comment by ThyWoof** [comment]
https://github.com/ChrisPJohn/SolastaCommunityExpansion/issues/703

File: SolastaCommunityExpansion/Models/ItemOptionsContext.cs [hunk 1]
```diff
@@ -105,7 +127,9 @@ private static ItemDefinition CreateAndAddToDB(string name, string guid, string
                 "Equipment/&DruidicAmuletDescription",
                 ComponentPouch_ArcaneAmulet,
                 EquipmentDefinitions.FocusType.Druidic,
-                BeltOfGiantHillStrength.GuiPresentation.SpriteReference);
+                BeltOfGiantHillStrength.GuiPresentation.SpriteReference,
+                EquipmentDefinitions.SlotTypeMainHand,
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Models/ItemOptionsContext.cs:131]: I don't think we want the amulet to slot into hands.

File: SolastaCommunityExpansion/Models/ItemOptionsContext.cs [hunk 2]
```diff
@@ -123,7 +149,9 @@ private static ItemDefinition CreateAndAddToDB(string name, string guid, string
                 "Equipment/&LivewoodStaffDescription",
                 ComponentPouch_ArcaneAmulet,
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Models/ItemOptionsContext.cs:150]: Should this be quarterstaff like the arcane staff above?

**Review by ChrisPJohn — CHANGES_REQUESTED** [review]

File: SolastaCommunityExpansion/Models/ItemOptionsContext.cs (hunk 1 above)
Comment (ThyWoof) [reply SolastaCommunityExpansion/Models/ItemOptionsContext.cs]: changed builder logic. fixed.

File: SolastaCommunityExpansion/Models/ItemOptionsContext.cs (hunk 2 above)
Comment (ThyWoof) [reply SolastaCommunityExpansion/Models/ItemOptionsContext.cs]: This was in original code when we migrated but makes sense to change to quarterstaff.

**Review by ChrisPJohn — APPROVED** [review]

**Review by PhilPJL — APPROVED** [review]

**Pull request merged**